### ISSUE TITLE
OpenCL Kernel Analyzer fails

### ISSUE THREAD

**Issue by robinchrist** [GPUOpen-Tools/CodeXL#210]
Hi,

I cannot analyze any OpenCL Kernels.
Even with no flags and no kernel content, the process fails for all Graphics IPs.

Output:
```
9> OpenCL1 for Hawaii: /home/robin/CodeXL_Linux_x86_64_2.5.25/rga

Target GPU detected:

Hawaii (Graphics IP v7)
	AMD FirePro W8100
	AMD FirePro W9100
	AMD Radeon (TM) R9 390 Series
	AMD Radeon R9 200 Series

Building for Hawaii... succeeded.
Error: failed to disassemble binary output and produce textual ISA code for Hawaii (kernel: OpenCL1).
Error: No IL for device 'Hawaii'.
Error: No IL for device 'Hawaii'.
Error: failed to disassemble binary output and produce textual IL code for Hawaii (kernel: OpenCL1).
```

HW:
Threadripper 1950X, Kubuntu 17.10
AMD R9 390X

UPDATE:
It works only for v6 IPs.

UPDATE:
It was a driver problem with AMDGPU-PRO 18.10, Works with 18.20 (however, nothing was mentioned in the release notes)

**Comment by tekcomm** [comment]
This is bionic
It is a 18.04 kubuntu based livecd that runs CodeXL and the latest firmware and mesa / amdgpu patches and firmware
CodeXL 2.5
![42119109-54afa462-7bf8-11e8-860b-e4ab8b1c1d39](https://user-images.githubusercontent.com/23721155/42139106-01997780-7dba-11e8-8e5e-0818d9b8d303.png)

I patched it up to work with beaver can you please test it out.
https://github.com/RadeonOpenCompute/ROCm/issues/361
I also added more tools and it is both a memory based livecd and kubuntu Bionic installer

**Comment by MorphBC** [comment]
Hey Tekcomm. Can I try ur liveCD??

**Comment by tekcomm** [comment]
Sure I'm building the.new Vega kernel right now

**Comment by bpurnomo** [comment]
It looks like based on the latest update from the reporter, this issue is resolved in 18.20.
